client-ts/Microsoft.AspNetCore.SignalR.Test.Server/wwwroot/js/common.js

```diff
@@ -1,22 +1,24 @@
 const ECHOENDPOINT_URL = `http://${document.location.host}/echo`;
 
+function getTransportTypes() {
+    let transportTypes = [ signalR.TransportType.WebSockets ];
```

Comment:
Does the order of the transports matter here? Otherwise you can just initialize the transportTypes array with both Websockets  and LongPolling